great_expectations/rule_based_profiler/parameter_builder/simple_date_format_string_parameter_builder.py

```diff
@@ -210,7 +211,7 @@ def _build_parameters(
         # Now obtain 1-dimensional vector of values of computed metric (each element corresponds to a Batch ID).
         metric_values = metric_values[:, 0]
 
-        nonnull_count: int = sum(metric_values)
+        nonnull_count: int = sum(metric_values) + NP_EPSILON
```

Comment:
This is no longer an `int`, right?

Reply:
@NathanFarmer Good catch -- fixed!  Thanks!